### PR TITLE
Investigate top contributors to a comp

### PR DESCRIPTION
**Pull request by pascoesean**
Barplots to help visualize what percent of certain observational categories are highly represented in the top contributing cells to certain components (specifically used here for component 13):

figureS4_louvain (1)

### CODE REVIEW

**Comment by pascoesean** [comment]
@aarmey @andrewram4287 I'm adding you both as reviewers on this pull request (and maybe a few others today/tomorrow) in case you get an extra moment to look over some of this stuff, but absolutely no time crunch on these things and I can definitely go over this stuff with Andrew next week (I just thought it would be easier to keep track of what I was doing on here)

**Review by aarmey — APPROVED** [review]
@pascoesean this all looks reasonable to me. Were you wanting me to check something in particular?

**Comment by pascoesean** [comment]
> @pascoesean this all looks reasonable to me. Were you wanting me to check something in particular?

nope! Was just getting approval before merge.

File: sccp/figures/figureS4.py
Comment (andrewram4287) [inline sccp/figures/figureS4.py:41]: @pascoesean What is the louvain clustering column mean? What is the benefit of looking at it rather than the cell type broad column?

Comment (andrewram4287) [reply sccp/figures/figureS4.py:41]: @pascoesean It would it make it easier if you could add more description of what each Louvain cluster ends up being for the cell type broad. It's difficult to know what you are referring to in these plots

also shouldn't the percents add up to one?

**Pull request merged**
